src/sentry/static/sentry/app/views/performance/transactionDetails/eventMetas.tsx

```diff
@@ -48,29 +48,44 @@ function EventMetas({event, organization, projectId, location, quickTrace}: Prop
   const httpStatus = <HttpStatus event={event} />;
 
   return (
-    <EventDetailHeader>
+    <EventDetailHeader type={type}>
       <MetaData
         headingText={t('Event ID')}
-        tooltipText={t('The unique ID assigned to this transaction.')}
+        tooltipText={t('The unique ID assigned to this %s.', type)}
         bodyText={getShortEventId(event.eventID)}
         subtext={projectBadge}
       />
-      <MetaData
-        headingText={t('Total Duration')}
-        tooltipText={t(
-          'The total time elapsed between the start and end of this transaction.'
-        )}
-        bodyText={getDuration(event.endTimestamp - event.startTimestamp, 2, true)}
-        subtext={timestamp}
-      />
-      <MetaData
-        headingText={t('Status')}
-        tooltipText={t(
-          'The status of this transaction indicating if it succeeded or otherwise.'
-        )}
-        bodyText={event.contexts?.trace?.status ?? '\u2014'}
-        subtext={httpStatus}
-      />
+      {isTransaction(event) ? (
+        <MetaData
+          headingText={t('Total Duration')}
+          tooltipText={t(
+            'The total time elapsed between the start and end of this transaction.'
+          )}
+          bodyText={getDuration(
+            transaction.endTimestamp - transaction.startTimestamp,
```

Comment:
You can use `event` here now that its been typed checked to be a `EventTransaction`.